### DISCUSSION TITLE
Welcome to CODEGENIE-2023!

### DISCUSSION THREAD

**Discussion by rparakhiya** [tark-ignite/codegenie-2023#1] [Announcements]
## 👋 Welcome!

This discussion board will be used to post announcements and to discuss questions contests may have.

### Useful links:
* [Event Information](https://tarktech.com/codegenie-2023)
* [Qualifier Round Result](https://tarktech.com/codegenie-2023/)
* [Practice Problem: Final Round](https://github.com/tark-ignite/codegenie-2023/discussions/7)
* [Common Tricks & Technique to solve a Programming/Algorithmic Problem](https://github.com/tark-ignite/codegenie-2023/discussions/6)
* [Practice Problem Statements](https://github.com/tark-ignite/codegenie-2023/raw/main/practice-problem-statements.pdf)

**Comment by rparakhiya** [comment]
Want to sharpen your weapons before the fight? Get your hands on to solve these [practice problem statements](https://github.com/tark-ignite/codegenie-2023/raw/main/practice-problem-statements.pdf)

**Reply by vimal-parakhiya** [reply]
Hello Codegenie Participants!

We hope that you have solved the _practice problem statments_ shared above. Do not hesitate and upload your solutions, even if you have cracked few of them. You may upload the solutions to your github repository and post the link on [this ](https://github.com/tark-ignite/codegenie-2023/discussions/3) page.

Our team will review them and get back to you with suggestions, if applicable, to make you ready for the upcoming programming rounds.

**Comment by rparakhiya** [comment]
Joining link is shared with all registrants via email. Please make sure to check junk/spam mailbox if you don't see it in your inbox. Feel free to reach out on the Q&A thread if you registered yourself and haven't received invitation email.

All the best!

**Comment by rparakhiya** [comment]
Hey Hackers! Results of the qualifier rounds are in! List of Top 20 winners of the qualifier round and selected participants of the final round is published on the event page: https://tarktech.com/codegenie-2023/

All winners and qualifiers will also be notified over email and phone.

Congratulations to all winners and qualifiers! 🎉
